src/picongpu/include/algorithms/AssignedTrilinearInterpolation.hpp

```diff
@@ -74,24 +74,21 @@ struct AssignedTrilinearInterpolation
         typedef typename ::PMacc::result_of::Functor<AssignedTrilinearInterpolation, Cursor>::type type;
 
         type result_z = type(0.0);
-#pragma unroll 4
-        for (float_X z = Begin; z <= End; z += float_X(1.0))
+        for (int z = Begin; z <= End; ++z)
```

Comment:
since you are touching the for loops anyway, feel free to make `their( spacings, pretty )`, there is not much other code in the file :)